app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_comment_counter
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [OPTIONAL] It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new post has been saved to database, then make the `update_comment_counter` method private. I can see that you've already used one active callback, but I would suggest to make the `update_comment_counter` method private.

Reply:
This change has been done well